### PR TITLE
Show a preview image for playlist items

### PR DESCRIPTION
**Pull request by donato**
[#92873620]

### CODE REVIEW

File: src/js/view/preview.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        },
+        loadImage: function(model, playlistItem) {
+            var img = playlistItem.image;
+
+            if (_.isString(img)) {
+                this.el.style['background-image'] = 'url(' + img + ')';
+            } else {
+                this.el.style['background-image'] = '';
```
Comment (jw-kaurand) [inline src/js/view/preview.js]: 'this' refers to the model when we get here as a result of the listener on the model.  It's causing errors when switching between playlist items.

Comment (donato) [inline src/js/view/preview.js]: Yup, you're right, line 10 should bind "this" by using the third parameter.
I'll get that in tomorrow.

File: src/js/view/view.js [hunk 2]
```diff
@@ -879,26 +880,13 @@ define([
...
-
             // debug this, find out why
-            if (!(_isMobile && model.fullscreen)) {
-                model.getVideo().setControls(false);
+            if (!(_isMobile && _model.fullscreen)) {
+                _model.getVideo().setControls(false);
             }
         }
 
         function _hideDisplay() {
```
Comment (jw-kaurand) [inline src/js/view/view.js]: Seems like we can get rid of this and a few of the calls to it now that there's nothing left in it

File: src/css/imports/display.less [hunk 3]
```diff
@@ -1,11 +1,20 @@
 @import "vars.less";
 .jw-display {
     position: absolute;
-    display: none;
+    //display: none;
     width: 100%;
     height: 100%;
 }
 
+/*
```
Comment (jw-kaurand) [inline src/css/imports/display.less]: Can these be removed or are they reminders for something?

**Comment by jw-kaurand** [comment]
A newer version of this branch has been merged.

**Pull request closed without merging**